templates/blocks/share_buttons.twig

```diff
@@ -1,23 +1,28 @@
 {% block javascripts %}
     <script>
-            window.addEventListener('DOMContentLoaded', () => {
-                // If the native share functionality is available on the device, we show the corresponding share buttons.
-                if (!navigator.share) {
-                    return;
-                }
-
-                const nativeShareButtons = document.querySelectorAll('.share-buttons .share-btn.native');
-                nativeShareButtons.forEach(nativeShareButton => nativeShareButton.style.display = 'block');
-            });
+        window.addEventListener('DOMContentLoaded', () => {
+            if (typeof dataLayer === undefined) {
```

Comment:
I am not 100% sure about this, but I think this will always evaluate to false because `typeof` returns a string, so the correct way to check if `dataLayer` is undefined would be:
```js
if (typeof dataLayer === 'undefined') {
    return;
}
```

Reply:
Funny that you say that, because I just had this exact same issue in another PR of mine, and I think you're right! I'll update the code

Reply:
Although the share functions are still working so that confuses me 🤔